LFGBulletinBoard/Options.lua

```diff
@@ -219,8 +219,18 @@ local function GenerateExpansionPanel(expansionID)
 	end
 	
 	if not isClassicEra then
-		CheckBoxChar("HeroicOnly", false)
-		CheckBoxChar("NormalOnly", false)
+		local heroicOnly = CheckBoxChar("HeroicOnly", false)
+		local normalOnly = CheckBoxChar("NormalOnly", false)
+		local exclusiveUpdateHandler = function(updatedBox, mutualBox)
```

Comment:
same logic used to handle the `ChatStyle` vs `CompactStyle` options below starting on line 345